cypress.config.ts

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  // setupNodeEvents can be defined in either
+  // the e2e or component configuration
+  e2e: {
+    setupNodeEvents(on, config) {
+      // modify config values examples
+      // config.defaultCommandTimeout = 10000
+      video: false,
+      screenshotOnRunFailure: false,
+
+      // IMPORTANT return the updated config object
+      return config;
+    },
+  },
+});
```

Comment:
this config file is not needed as you are using `cypress.json` to setup the config of cypress.